fv3core/pace/fv3core/initialization/analytic_init.py

```diff
@@ -0,0 +1,66 @@
+from enum import Enum, EnumMeta
+
+import pace.util as fv3util
+from pace.fv3core.dycore_state import DycoreState
+from pace.util.grid import GridData
+
+
+class MetaEnumStr(EnumMeta):
+    def __contains__(cls, item):
+        return item in cls.__members__.keys()
+
+
+class Cases(Enum, metaclass=MetaEnumStr):
+    baroclinic = "baroclinic"
+    tropicalcyclone = "tropicalcyclone"
+
+
+def init_analytic_state(
+    analytic_init_str: str,
```

Comment:
I still think it would be better to call this `case` instead  of `analytic_init_str` but won't block it

Reply:
I am sorry I missed this one. I will make the change now.